src/mlpack/methods/preprocess/CMakeLists.txt

```
@@ -48,6 +48,13 @@ add_go_binding(preprocess_scale)
 add_julia_binding(preprocess_scale)
 add_markdown_docs(preprocess_scale "cli;python;julia;go" "preprocessing")
 
+add_cli_executable(preprocess_one_hot_encoding)
+add_python_binding(preprocess_one_hot_encoding)
+add_go_binding(preprocess_one_hot_encoding)
+add_julia_binding(preprocess_one_hot_encoding)
+add_markdown_docs(preprocess_one_hot_encoding "cli;python;julia;go"
+  "preprocessing")
```

```suggestion
    "preprocessing")
```
Just a little style fix. :)